worker-pools.yml

```diff
@@ -2958,6 +2958,40 @@ pools:
               type: SCRATCH
               interface: NVME
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-docker'
```

Comment:
I like the naming! I assume `-docker` implies that `disableNativePayloads` should be `true`?

Reply:
Yup.

Reply:
(It is, as far as I can tell)

Reply:
This makes me wonder if we ought to have an implementation for these types of pools that would help enforce this (it wouldn't be a guarantee, but it would make it more difficult to forget to set it...).